README.md

```diff
@@ -13,21 +13,34 @@ The Open Grid Emissions Dataset can be [downloaded here](https://singularity.ene
 
 To install and run the pipeline on your computer, open anaconda prompt, navigate to the folder where you want to save the repository, and run the following commands:
 
-```
+```bash
 conda install git
 git clone https://github.com/singularity-energy/open-grid-emissions.git
 conda update conda
 cd open-grid-emissions
 conda env create -f environment.yml
 conda activate open_grid_emissions
+```
+A more detailed walkthough of these steps can be found below in the "Development Setup" section.
+
+It is also possible to use `pipenv`:
+```bash
+pipenv sync
```

Comment:
Does the user first need to `cd` to a specific directory before running this?

Reply:
I will clarify

Reply:
The code snippet should include all the steps now